README.md

```diff
@@ -34,6 +34,24 @@ There is a free plan.
 Heroku uses [gunicorn](http://flask.pocoo.org/docs/dev/deploying/wsgi-standalone/#gunicorn)
 to run the server, see the [Procfile](Procfile).
 
+### Docker
+#### Docker build
```

Comment:
This is a nice addition and I would already merge this.

I have some ideas for how I personally like it better and I will comment on it. But since it's personal and this is a cool addition, I am willing to merge it fast.